src/pages/rooms/RoomDetails.page.vue

```diff
@@ -8,7 +8,7 @@
 	>
 		<template slot="header">
 			<div class="ma-2">
-				<div class="text-h3 pb-2 course-title">
+				<div class="text-h3 pb-2 course-title" data-testid="courses-course-title">
```

Comment:
Prettier möchte es gerne so formattiert haben, damit die Zeile nicht zu lang wird.

```suggestion
				<div
					class="text-h3 pb-2 course-title"
					data-testid="courses-course-title"
				>
```